mono/metadata/assembly.c

```diff
@@ -254,8 +270,8 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.Aes"),
 	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.ECDiffieHellman"),
 	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.ECDsa"),
-	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.Hashing"),
-	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.Hashing.Algorithms"),
+	FACADE_ASSEMBLY ("System.Security.Cryptography.Hashing"),
+	FACADE_ASSEMBLY ("System.Security.Cryptography.Hashing.Algorithms"),
```

Comment:
what happened here?  Were the ones with `Encryption` in the name incorrect?

Reply:
yup, it's the namespace, not the assembly name ;)